### DISCUSSION TITLE
Error in mubu.maxhelp copy/paste tab

### DISCUSSION THREAD

**Discussion by mattyostrowski** [Ircam-RnD/catart-mubu#115] [Bug, Issues, and Suggestions]
in the copy/paste tab of the MuBu maxhelp file, on the top row are message boxes "track A copy", etc. The track that they point to is called 'copybpf', however, so you get an error: 

imubu: invalid track name 'A' in track message

Also, in the row below, the demo buffer you are supposed to copy from is empty.

**Comment by RiccardoBorghesi** [comment]
Thanks Mat, will be fixed in the next release.

**Comment by mattyostrowski** [comment]
<div></div><div>Thanks Mat, will be fixed in the next release.
<br/>
<br/>-- 
<br/>Reply to this email directly or view it on GitHub:
<br/>https://github.com/ircam-ismm/catart-mubu/discussions/115#discussioncomment-8632451
<br/>You are receiving this because you authored the thread.
<br/>
<br/>Message ID: ***@***.***&gt;</div>
